app/src/main/java/com/muratozturk/mova/common/Extensions.kt

```diff
@@ -308,6 +308,15 @@ fun Context.openShareIntent(text: String) {
     )
 }
 
+/**
+ * Opening Url on Device's Browser
+ * @param webPageUrl the url which will navigate
+ */
+fun Context.openWebPage(webPageUrl: String) =
+    Intent(Intent.ACTION_VIEW)
+        .apply { data = Uri.parse(webPageUrl) }
+        .also(::startActivity)
```

Comment:
This is good looking code but a little misuse of Extension functions. Opening a webPage on browser would be better as a part of a different class such as "WebUtil". Which explains very well what's its doing. 
Opening another browser is not realated to Context, so using Extension function won't be a good choice.